library/src/nvidia_detail/hipblas.cpp

```diff
@@ -226,6 +226,57 @@ cudaDataType_t HIPDatatypeToCudaDatatype(hipblasDatatype_t type)
     }
 }
 
+cudaDataType_t HIPDatatypeToCudaDatatype_v2(hipDataType type)
+{
```

Comment:
Can this be `HIPDatatypeToCudaDatatype` rather than `HIPDatatypeToCudaDatatype_v2` ?

Reply:
No, the original, which takes a hipblasDatatype_t, is called (mistakenly?) `HIPDatatypeToCudaDatatype` so can't overload it.